client/src/pages/game/GamePage.tsx

```diff
@@ -23,13 +81,36 @@ export default function GamePage() {
   const onSubmitGuess = (event: React.FormEvent<HTMLFormElement>) => {
     event.preventDefault();
     setGuesses([...guesses, guess]);
+    setGuess("");
+  };
+
+  const renderDropdownHint = (): IDropdownMenuItems[] => {
+    if (!guess || !words) return [];
+
+    return words
+      .filter((word) => {
+        const wData = word.word_data;
+        const name = wData.find(
+          (header) => header.header_name === "Name"
+        )?.value;
+        return name?.toLowerCase().startsWith(guess.toLowerCase());
+      })
+      .map((word) => {
+        const wData = word.word_data;
+        const name =
+          wData.find((header) => header.header_name === "Name")?.value || "";
+        return {
+          name,
+          onClick: () => setGuess(name),
+        };
+      });
```

Comment:
the way the data comes in im concerned with a few things here:

- first this assumes there willl always be a name column, so maybe validate that or order the columns so first column is always identifier
- if ordered i can the .finds here and i can just grab first column
-